### PR TITLE
Add note on samples

### PR DESCRIPTION
**Pull request by usha-mandya**
Add a note to state the samples are not for production use.

### CODE REVIEW

**Comment by usha-mandya** [comment]
@aiordache Could you PTAL?

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 These samples provide a starting point for how to integrate different services using a Compose file and to manage their deployment with Docker Compose.
 
+> **Note:**
+>
+> The following samples use the build in development server to run your application. Do not use these in a production environment.
```
Comment (aiordache) [inline README.md:11]: I find the first sentence a little bit confusing. How about: 
`The following samples are intended for use in local development environments (project setups, tinkering with software stacks etc). The samples are not to be deployed in production environments.`

Comment (usha-mandya) [reply README.md]: Sounds good. Updated the text with minor changes

**Comment by glours** [comment]
@usha-mandya can you sign your commit?
`git commit --amend -s --no-edit` and the `push --force`?

**Comment by usha-mandya** [comment]
@glours Done. Thanks for the tip!

**Comment by AleksMeshkov** [comment]
I wonder why I can't use it in production?

**Pull request merged**